### DISCUSSION TITLE
From DevOps to Github?

### DISCUSSION THREAD

**Discussion by mschmicking** [wulfland/BootCampBasta23#11] [Q&A]
# Question

@wulfland what do you think makes more sense? Keeping running env. on DevOps or moving on to GitHub?

| # | DevOps | GitHub |
|--------|--------|--------|
| Repos | ✅  | ✅  |
| Pipelines / Actions| ✅  | ✅  |
| etc... | Cell | Cell | 

``` JS
console.log("Hello Basta 2023");
```

<details><summary>Details</summary>
<p>

This is new to me 😄 

</p>
</details>

**Comment by wulfland** [comment]
You mean environments in pipelines, @0x7878 ?

**Reply by mschmicking** [reply]
Sorry for the late response. I meant the environment /ecosystem in general. In fact both products are owned by Microsoft and are quite similar. Right now we use DevOps but I like GitHub more.
